cmd/copy.go

```diff
@@ -1572,6 +1572,17 @@ func (cca *CookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 		err = e.enumerate()
 
 	case common.EFromTo.BlobFSTrash():
+		if cca.Recursive { // Extra hacky hack: Carrying recursive into STE temporarily to enable cancel of delete job
+			query, err := url.ParseQuery(jobPartOrder.SourceRoot.ExtraQuery)
```

Comment:
Is this because the delete API doesnt allow you to set recursive as a parameter?

Reply:
No, this is because I felt like it did not make sense to tack a whole additional property onto the job plan file format to carry through the recursive option for ADLS Gen 2.

Reply:
If we wanted to handle proper enum/delete then this could be completely removed.